### DISCUSSION TITLE
The graphics in Topology not refreshed with appStore.durationTime updated

### DISCUSSION THREAD

**Discussion by CzyerChen** [apache/skywalking#11963] [Q&A] [answered]
Case: I changed the durationTime at the top right, and the data in Topology updated, but the graphics not updated. 
What I want: When I change the the durationTime at the top right, the data refreshed, and then the graphics should be updated too.

I found the code in `ServiceMap.vue`

```
 watch(
    () => appStore.durationTime,
    () => {
      if (dashboardStore.entity === EntityType[1].value) {
        freshNodes();
        hierarchyRelated.value = false;
      }
    },
  );
```

`EntityType[1].value = ALL` && `dashboardStore.entity = Service`
<img width="1126" alt="Pasted Graphic" src="https://github.com/apache/skywalking/assets/22817918/a773025d-dcbe-4959-bc65-58c9eaf01344">

Why is the type of  entity`ALL` freshNodes, but the type of Topology dashboard `Service` not.

**Comment by wu-sheng** [comment]
@Fine0830 Please confirm, this seems a bug. Both type of topology should work.

**Comment by wu-sheng** [comment, marked as answer]
Please follow https://github.com/apache/skywalking-booster-ui/pull/374
